app/src/main/java/com/sun/wsm_android_v2/data/model/TimesheetTotal.kt

```diff
@@ -1,3 +1,8 @@
 package com.sun.wsm_android_v2.data.model
 
-class TimesheetTotal(var symbol: String?, var title: String?, var timeUnit: String?, var totalNumber: Number?)
+data class TimesheetTotal(
+    var symbol: String = "",
+    var title: String = "",
+    var timeUnit: String = "",
+    var totalNumber: Number = 0.0
```

Comment:
chuyển gíup a sang kiểu Double đi e
```suggestion
    var totalNumber: Double = 0.0
```